lib/common.go

```diff
@@ -187,16 +199,20 @@ func BigIntToBytes(alph Alphabet, x *big.Int, l int) []byte {
 
 	// the length must be the same as the original byte
 	// so we add extra zeros to the end if necessary
+	overflow := false
```

Comment:
It would be more "Single Responsibility"-ish to have a separate function for determining the overflow/carry 😛